cluster-scope/components/resourcequotas/large/resourcequotas.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: large
+spec:
+  hard:
+    requests.cpu: '8'
+    requests.memory: 40Gi
+    limits.cpu: '8'
+    limits.memory: 40Gi
+    persistentvolumeclaims: '8'
+    requests.storage: 80Gi
```

Comment:
@HumairAK I think we can give out more storage and less memory

Reply:
Any suggestions on specific values? And are you referring to just `large` or all?